mesheryctl/pkg/utils/healthcheck.go

```diff
@@ -161,6 +161,7 @@ func IsMesheryRunning(currPlatform string) (bool, error) {
 			deploymentList, err := deploymentInterface.List(context.TODO(), metav1.ListOptions{})
 
 			if err != nil {
+				err = errors.New("Meshery Server unavailable. Is your Meshery Server up and running?\n\nUse `mesheryctl system status` to confirm or switch contexts.")
```

Comment:
Can you also use a meshkit error see available example 

ex: https://github.com/meshery/meshery/blob/5e7e59a2de62d45f3ba9aa8e89a832dc4f09a059/mesheryctl/internal/cli/root/error.go#L47